tools/mass_perldoc_md_creation.pl

```diff
@@ -70,6 +70,7 @@ =head1 AUTHORS
     'tools/MakeArchiveLocationRelative.pl',
     'tools/ProdToConfig.pl',
     'tools/database_files_update.pl',
+    'tools/delete_mri_upload.pl',
```

Comment:
delete_mri_upload.pl should be replaced by delete_imaging_upload.pl here.

Reply:
Ooops...Forgot about that one. Done.